sunyar-fabric.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+title: Sunyar fabric
+parent: Labs
+---
+# Lab Name
+Sunyar fabric
+
+# Short Description
+Sunyar is a settlement distributed system based on Hyperledger fabric. This is a non-profit project for contributing charities to help needy people in a decentralized ecosystem.
+
+# Scope of Lab
+Sunyar is a settlement distributed system based on Hyperledger fabric. There are several usecased in banking usecased in settlement distributed systems. Our goal is to achieve an appropriate project on hyperledger fabric to solve other challenges and usecases on banking systems.
```

Comment:
```suggestion
Sunyar is a settlement distributed system based on Hyperledger Fabric. There are several use cases in banking. Our goal is to achieve an appropriate project on Hyperledger Fabric to solve other challenges and use cases on banking systems.
```

Reply:
From the README in the repo linked below, there is the following description:

> Sunyar is a distributed settlement platform, which can be used by charities for managing donation process. This can be done by an open-source platform, which can link to a distributed ledger for charities. The open-source platform helps them for automating their process and the distributed ledger can used for two main reasons: uniqueness of needy and their requirements, and transparent aid for them).

Based on the scope given above, it looks like the intent is to go beyond just the charity aspect. Is this true?

Reply:
Actually our Lab has focused on distributed settlement to accomplish other use cases in Fintech and banking systems such as Post-trade securities and Cross border payments.